frontend/pages/policies/ManagePoliciesPage/components/CalendarEventsModal/CalendarEventsModal.tsx

```diff
@@ -40,7 +40,7 @@ interface ICalendarEventsModal {
   enabled: boolean;
   url: string;
   policies: IPolicyStats[];
-  gomEnabled?: boolean;
+  gitOpsModeEnabled?: boolean;
```

Comment:
I'm not sure its worth putting in the changes for these four modals, as I'll have to back them completely out and add them to the PoliciesPaginatedList instead 🫤

Reply:
Okay, I'll back those changes out now